app/otr/client/rocketchat.otr.room.js

```diff
@@ -67,6 +68,7 @@ OTR.Room = class {
 	}
 
 	end() {
+		APIClient.v1.post('statistics.telemetry', { params: [{ eventName: 'otrStats', timestamp: Date.now(), rid: this.roomId }] });
```

Comment:
We need to send the event on the acknowledge step, as the `end` step is not always executed.